src/harness/testcases/testdata/esc_zone_contains_device_a.json

```diff
@@ -1,10 +1,18 @@
-{"id" :"zone/exclusion_zone/ntia/2017_04_10/esc_zone_contains_deviceA_harness_test",
-			 "creator" : "SAS_Administrator_Test",
-			 "usage" : "EXCLUSION_ZONE",
-			 "zone" :
-			 {"type": "Polygon","coordinates": [
-						 [
-						 [37.525056, -122.284112],
-						 [37.125056, -122.284112],[37.125056, -121.884112],[37.525056, -121.884112]]]
-			}
+{
+ "id" :"zone/exclusion_zone/ntia/2017_04_10/esc_zone_contains_deviceA_harness_test",
```

Comment:
ESC zones aren't created by the NTIA so I'm not sure this is a good format for the ID. Maybe its best to say that this field will not be filled in by the test harness (instead the SAS under test will generate the ID) since there's no standard ID format for ESCs?